litmus-portal/frontend/src/views/ChaosWorkflows/BrowseCluster/styles.ts

```diff
@@ -144,6 +144,25 @@ const useStyles = makeStyles((theme) => ({
   datePickerColor: {
     color: theme.palette.secondary.dark,
   },
+  // Table status
+  completed: {
+    width: '5.9125rem',
+    textAlign: 'center',
+    borderRadius: 3,
+    paddingTop: theme.spacing(0.375),
+    paddingBottom: theme.spacing(0.375),
+    color: theme.palette.primary.dark,
+    background: theme.palette.customColors.menuOption.active,
+  },
+  failed: {
+    width: '5.9125rem',
+    textAlign: 'center',
+    borderRadius: 3,
+    paddingTop: theme.spacing(0.375),
+    paddingBottom: theme.spacing(0.375),
+    color: theme.palette.error.dark,
+    backgroundColor: theme.palette.error.light,
+  },
```

Comment:
CSS for both completed and failed are almost same...can you please follow `${}${}` format